### PR TITLE
HPCC-14052 Fix error on implicit project of MERGE(,SORTED(childrow.field))

### PR DESCRIPTION
**Pull request by ghalliday**
Signed-off-by: Gavin Halliday gavin.halliday@lexisnexis.com

### CODE REVIEW

**Comment by ghalliday** [comment]
@jamienoss please review

File: ecl/regress/issue14052.ecl [hunk 1]
```diff
@@ -0,0 +1,54 @@
+layout1 := {
```
Comment (jamienoss) [inline ecl/regress/issue14052.ecl]: These code looks familiar, (regarding the comments) is it cannibalised (just curious)?

**Comment by jamienoss** [comment]
@richardkchapman this looks good to me.

**Comment by richardkchapman** [comment]
Missed the boat for 5.4.4 - can you retarget to 5.4.6 or 6.0.0 as appropriate

**Comment by richardkchapman** [comment]
@ghalliday Please retarget

**Pull request closed without merging**